Added file size info to admin image box, added file size errors

jrtaylor-com opened the pull request:
This file size takes into account the added 1.333 bytes per byte that the base64 encode that is used on the backend adds

kenwalker commented on orkui/controller/controller.Admin.php:
```diff
@@ -682,15 +682,20 @@ public function player($id) {
-										'Token' => $this->session->token
-									));
+								if ((int) $_FILES['Heraldry']['size'] * 1.333 > 465000) {
+									$this->data['Error'] = 'Image Error: File size is to large.';
```
"too large" same with one below

kenwalker commented on orkui/template/default/Admin_player.tpl:
```diff
@@ -25,6 +25,14 @@
+		</div>
+		<div>
+			<span></span>
+			<span style="font-size: 10px;">Max Size: 2725 Kb (348836 bytes)</span>
```
Most people are not familiar with Kilobits and most file information within OSs is shown in KB or MB. I suggest changing to KB and omitting bytes all together.

kenwalker commented:
Have left some comments above but otherwise the functionality works. It's a bit weird that you get both a green "Player updated" and the "Image error".  Not a big deal though.
<img width="459" alt="image" src="https://user-images.githubusercontent.com/1138820/115148242-b1fbb380-a02c-11eb-8d57-f8c849ab0ba5.png">

jrtaylor-com commented:
@kenwalker I added the spelling mods, and modified the image size text. I kept Bytes since both OSs show bytes when inspecting a file. Converted kilobits to kilobytes.

![Screen Shot 2021-04-18 at 9 20 23 AM](https://user-images.githubusercontent.com/2745760/115149165-7232cd00-a028-11eb-9bd9-0e6a582d9b8c.png)
<img width="494" alt="Screen Shot 2021-04-18 at 9 19 39 AM" src="https://user-images.githubusercontent.com/2745760/115149167-72cb6380-a028-11eb-90ee-b7877131d368.png">

jrtaylor-com commented:
@kenwalker added officers to kingdom/park index page 
![Screen Shot 2021-04-18 at 10 21 51 AM](https://user-images.githubusercontent.com/2745760/115150883-ec1a8480-a02f-11eb-9eca-26c9a48e85a3.png)
![Screen Shot 2021-04-18 at 10 21 41 AM](https://user-images.githubusercontent.com/2745760/115150885-ecb31b00-a02f-11eb-8281-cac03645423f.png)

kenwalker commented:
Monarchy info on both pages looks good

kenwalker commented:
There are approximately 39 parks (according to the sql query) that have not put http:// or https:// in front of their URL and in that case your link does not work properly. For example, try Felfrost's link.

jrtaylor-com commented:
> There are approximately 39 parks (according to the sql query) that have not put http:// or https:// in front of their URL and in that case your link does not work properly. For example, try Felfrost's link.

I feel like once they see the feature they might fix their links. They didn't have a reason to before.

kenwalker commented:
> > There are approximately 39 parks (according to the sql query) that have not put http:// or https:// in front of their URL and in that case your link does not work properly. For example, try Felfrost's link.
> 
> I feel like once they see the feature they might fix their links. They didn't have a reason to before.

I volunteer as tribute to just go fix them ha.  I wonder if we could add a validation to the setting of it for any future entries to ensure it has http:// or https:// ?

jrtaylor-com commented:
> > > There are approximately 39 parks (according to the sql query) that have not put http:// or https:// in front of their URL and in that case your link does not work properly. For example, try Felfrost's link.
> > 
> > 
> > I feel like once they see the feature they might fix their links. They didn't have a reason to before.
> 
> I volunteer as tribute to just go fix them ha. I wonder if we could add a validation to the setting of it for any future entries to ensure it has http:// or https:// ?

I expect a lot of them to be outdated links anyhow. the http/https would be dependent on the host themselves, not on the ORK server. You could replace them with `//` instead of `https:` or `http:` which would just use the ORK's current state. This would keep them from throwing a mixed content error when the ORK is using https.

Only issue would be if the target server doesn't handle switching between http/https gracefully

kenwalker commented:
> > > > There are approximately 39 parks (according to the sql query) that have not put http:// or https:// in front of their URL and in that case your link does not work properly. For example, try Felfrost's link.
> > > 
> > > 
> > > I feel like once they see the feature they might fix their links. They didn't have a reason to before.
> > 
> > 
> > I volunteer as tribute to just go fix them ha. I wonder if we could add a validation to the setting of it for any future entries to ensure it has http:// or https:// ?
> 
> I expect a lot of them to be outdated links anyhow. the http/https would be dependent on the host themselves, not on the ORK server. You could replace them with `//` instead of `https:` or `http:` which would just use the ORK's current state. This would keep them from throwing a mixed content error when the ORK is using https.
> 
> Only issue would be if the target server doesn't handle switching between http/https gracefully

I mean I’ll try them out individually and fix their info.
